packages/gasket-plugin-intl/lib/build-modules.js

```diff
@@ -169,14 +177,57 @@ class BuildModules {
     }, Promise.resolve([]));
   }
 
+  /**
+   * Find modules with locale directories to process
+   *
+   * @returns {SrcPkgDir[]} source package directories
+   */
+  async gatherModuleDirs() {
+    if (this._lookupModuleDirs) {
+      const promises = this._lookupModuleDirs.map(async lookupDir => {
+
+        const match = lookupDir.match(rePkgParts);
+        if (!match?.groups?.name) {
+          this._logger.warning(`build:locales: malformed module name: ${lookupDir}`);
+          return;
+        }
+
+        const pkgName = match.groups.name;
```

Comment:
This might be a bit DRYer

```suggestion
        const pkgName = match?.groups?.name;
        if (!pkgName) {
          this._logger.warning(`build:locales: malformed module name: ${lookupDir}`);
          return;
        }
```